pandas/core/indexes/interval.py

```diff
@@ -546,7 +546,11 @@ def from_tuples(cls, data, closed='right', name=None, copy=False):
 
     def to_tuples(self):
         """Return an Index of tuples of the form (left, right)"""
```

Comment:
I guess would be ok to add a kwarg ``na_tuple=False`` here to select behavior as @shoyer suggests.